src/stats.c

```diff
@@ -913,23 +967,23 @@ int stats_getOutfallStat(int index, TOutfallStats *outfallStats)
 		memcpy(outfallStats, &OutfallStats[k], sizeof(TOutfallStats));
 		
 		// Perform Deep Copy of Pollutants Results
-		if (Nobjects[POLLUT] > 0)
-		{
-			outfallStats->totalLoad =
-				(double *)calloc(Nobjects[POLLUT], sizeof(double));
-			if (!outfallStats->totalLoad)
-			{
-				errorcode = ERR_MEMORY;
-			}
-			if (errorcode == 0)
-			{
-				for (k = 0; k < Nobjects[POLLUT]; k++)
-					outfallStats->totalLoad[k] = OutfallStats[k].totalLoad[k];
-			}
-		}
-		else outfallStats->totalLoad = NULL;
-	}
-	return errorcode;
+        if (Nobjects[POLLUT] > 0)
```

Comment:
@katmratliff, great bug squash! Thanks for catching this!  15 cool points awarded to you
👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 

🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛 🐛